ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -1191,7 +1200,10 @@ export class VaultMaxiProgram extends CommonProgram {
         if (this.isSingleMint && !mainAssetAsLoan) { //if there is a loan of the main asset, first pay back the loan
             let token = balances.get(this.mainCollateralAsset)
             if (token) {
-                collTokens.push(token)
+                console.log("cleanup to collateral "+token.amount+"@"+token.symbol)            
+                if(+token.amount > 0.1) { //main collateralAsset is DFI or DUSD. don't cleanup less than 10cent or 0.1 DFI
```

Comment:
maybe we should move % and 0.1 to an own variable? if we want to adapt it to not miss a place